### PR TITLE
Add build status and PyPI badges to README.md

### PR DESCRIPTION
**Pull request by MarcCote**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,4 +1,4 @@
-# TextWorld
+# TextWorld [![Build Status](https://travis-ci.org/Microsoft/TextWorld.svg?branch=master)](https://travis-ci.org/Microsoft/TextWorld) [![PyPI version](https://badge.fury.io/py/textworld.svg)](https://badge.fury.io/py/textworld)
```
Comment (tavianator) [inline README.md]: I think people usually put these on a new line under the title

Comment (MarcCote) [reply README.md]: You are right. I'll move them.

**Comment by MarcCote** [comment]
Done

File: README.md [hunk 2]
```diff
@@ -1,4 +1,5 @@
 # TextWorld
+[![Build Status](https://travis-ci.org/Microsoft/TextWorld.svg?branch=master)](https://travis-ci.org/Microsoft/TextWorld) [![PyPI version](https://badge.fury.io/py/textworld.svg)](https://badge.fury.io/py/textworld)
```
Comment (tavianator) [inline README.md]: Probably want another newline before "A text-based game generator..."  (https://github.com/Microsoft/TextWorld/blob/f65997b9868fb27cfbe91922c00befbcf0ad7073/README.md)

**Comment by MarcCote** [comment]
Alright, it should be good now.
https://github.com/Microsoft/TextWorld/blob/0c81bb0184dcb6152777c0c67b3ed56e85080fdf/README.md

**Pull request merged**